charts/seed-bootstrap/charts/loki/values.yaml

```diff
@@ -32,22 +32,22 @@ livenessProbe:
   httpGet:
     path: /ready
     port: metrics
-  initialDelaySeconds: 45
+  initialDelaySeconds: 120
 
 readinessProbe:
   httpGet:
     path: /ready
     port: metrics
-  initialDelaySeconds: 45
+  initialDelaySeconds: 120
 
 replicas: 1
 
 resources:
   limits:
-    cpu: 100m
+    cpu: 300m
     memory: 256Mi
   requests:
-    cpu: 20m
+    cpu: 200m
```

Comment:
difference between cpu requests and memory is 100m, and minChange is also 100m. If the pod goes into continuous restarts because of cpu throttling, VPA will not be able to recommend higher recommendations.
Should we increase `limits` to `450m` just to be on safe side?. Same for the memory as well.

Reply:
Did the change. Can you check please if it is better now?